en/development/exceptions.rst

```diff
@@ -64,7 +64,7 @@ exceptions for HTTP methods
 
 .. php:exception::UnauthorizedException
```

Comment:
@antograssiot Thanks!  Sorry I missed it out.  Forgot to build and check in my local first.